dev/packages/alpha/cisco/0.0.2/dataset/ios/agent/stream/pipeline.js

```diff
@@ -0,0 +1,222 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
```

Comment:
This file should not be in the package as we can't ship this yet through the elastic agent.

@andrewkroh @ph  This is interesting. I wonder how we will ship something like this down to Filebeat in the future. Easy answer would be if it just becomes part of the atomic config file.

For now, I assume this file is existing in Filebeat and we don't need it in the package?

Reply:
It would be ideal for maintainability to be able to keep these files as standalone .js files.

I wonder if this can be handled at some stage in the packaging or distribution. Like instead of sending this to the edge

```
  - script:
      lang: javascript
      id: cisco_ios
      file: ${path.home}/module/cisco/ios/config/pipeline.js
```

we evaluate the template and send the output of (i.e. inline the JS inside of the YAML):
```
  - script:
      lang: javascript
      id: cisco_ios
      source: >
{{ file("pipeline.js") | indent(6) }}
```

Reply:
Yes. @mtojek started to do something similar for field docs in the README's here: https://github.com/elastic/integrations/tree/master/dev/import-beats-resources/mysql/docs#error